smoke-tests-otel-starter/src/test/java/io/opentelemetry/smoketest/OtelSpringStarterSmokeTest.java

```diff
@@ -157,7 +168,19 @@ AutoConfigurationCustomizerProvider propagatorCustomizer() {
     }
   }
 
+  @AfterEach
+  void tearDown() {
+    reset();
+  }
+
+  private static void reset() {
+    SPAN_EXPORTER.reset();
+    METRIC_EXPORTER.reset();
+    LOG_RECORD_EXPORTER.reset();
+  }
+
   @Test
+  @org.junit.jupiter.api.Order(10)
```

Comment:
any reason this can't be imported?

Reply:
a spring `@Order` annotation is also used in this file